qemu/tests/cdrom.py

```diff
@@ -633,6 +633,10 @@ def test(self):
             serial_num = generate_serial_num()
             cdrom = params.get("cdroms", "").split()[-1]
             params["drive_serial_%s" % cdrom] = serial_num
+            cd_format = params.object_params(cdrom).get("cd_format")
+            if cd_format != "virtio_blk":
```

Comment:
Only virtio_blk not support wwn, so here should be cdrom != "virtio_blk" then add blk_extra_params to device.